### PR TITLE
formats tools

### PR DESCRIPTION
**Pull request by jburel**
Add bit to supported pixel type (used by OMERO)

### CODE REVIEW

File: components/formats-api/src/loci/formats/FormatTools.java [hunk 1]
```diff
@@ -105,11 +103,14 @@
   /** Identifies the <i>DOUBLE</i> data type used to store pixel values. */
   public static final int DOUBLE = 7;
 
+  /** Identifies the <i>DOUBLE</i> data type used to store pixel values. */
```
Comment (rleigh-dundee) [inline components/formats-api/src/loci/formats/FormatTools.java]: Comment is incorrect (double)

File: components/formats-api/src/loci/formats/FormatTools.java [hunk 2]
```diff
@@ -50,13 +50,11 @@
 import loci.formats.meta.MetadataStore;
 import loci.formats.services.OMEXMLService;
 import loci.formats.services.OMEXMLServiceImpl;
-
```
Comment (rleigh-dundee) [inline components/formats-api/src/loci/formats/FormatTools.java]: Does the whitespace need removing here?  imports from different components are usually separated in bioformats from what I can see.

File: components/formats-api/src/loci/formats/FormatTools.java [hunk 3]
```diff
@@ -1323,7 +1328,7 @@ public static void convert(IFormatReader input, IFormatWriter output,
    */
   public static long[] defaultMinMax(int pixelType) {
     long min = 0 , max = 0;
-
+
```
Comment (rleigh-dundee) [inline components/formats-api/src/loci/formats/FormatTools.java]: Leading whitespace

**Comment by jburel** [comment]
--no-rebase

**Pull request merged**